core/flyout_button.js

```diff
@@ -122,10 +135,14 @@ Blockly.FlyoutButton.prototype.createDom = function() {
         'rx': 4, 'ry': 4},
       this.svgGroup_);
 
+  cssClass = this.isLabel_ ? 'blocklyFlyoutLabelText' : 'blocklyText';
+  if (this.isLabel_ && this.callback_) {
```

Comment:
I'm a little confused by this part. why are labels clickable? seems like a clickable label is a button :)